### DISCUSSION TITLE
Aula - Classes & Objetos

### DISCUSSION THREAD

**Discussion by syspone** [balta-io/forum#271] [Tira Dúvidas] [answered]
@andrebaltieri , como vai?
Percebi que nessa aula o senhor comentou que são class existem os membros;

- Propriedades
- métodos
- eventos

Porem não vi o senhor comentando sobre `atributos` ! onde os atributos entram nesses conceitos de base de OO.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @cmsoouza, como vai?

Atributo é uma qualidade de um objeto, que no caso do C# (E java) pode ser transformado em uma propriedade.
Depois de gerado o objeto, as propriedades se tornam atributos e representam o estado do objeto.
